helm/charts/determined/templates/master-config.yaml

```diff
@@ -45,6 +45,11 @@ data:
       {{- end  }}
       port: {{ .Values.db.port }}
       name: {{ .Values.db.name | quote }}
+      {{- if .Values.db.sslMode }}
+      ssl_mode: {{ .Values.db.sslMode }}
+      {{- $rootCert := (required "A valid .Values.db.sslRootCert entry required!" .Values.db.sslRootCert )}}
```

Comment:
when using `sslMode`, are `certResourceName` and `resourceType` also required? should we add `required` statement for them as well?

or is there a way to inject and use the cert without these.

Reply:
yeah, I think we should already be checking for those in `master-deployment.yaml`